### ISSUE TITLE
Unable to compile using Xcode 12.5 and SwiftPM

### ISSUE THREAD

**Issue by mbanasiewicz** [Appboy/appboy-ios-sdk#275]
<img width="375" alt="Screenshot 2021-02-02 at 15 42 13" src="https://user-images.githubusercontent.com/1093407/106615970-4b0e4880-656d-11eb-9f9a-7df0806645f3.png">

**Comment by hokstuff** [comment]
Hi @mbanasiewicz,

Thanks for the heads up - Xcode 12.5 is currently in beta so it is possible there may be more changes. However, were you able to get `AppboyUI` working through SPM on your app before updating to Xcode 12.5 beta? If so, what versions of Xcode and our SDK did you upgrade from / to when you experienced this issue?

**Comment by vrutings** [comment]
Getting the same issue only when updating to Xdode 12.5

**Comment by salling** [comment]
Same issue in beta 2

**Comment by xav42mac** [comment]
Same problem here when updating to XCode 12.5 beta 2

**Comment by chshapiro** [comment]
Hey all,

We are able to repro these errors and are currently investigating. Xcode 12.5 is still in beta, and we will continue to keep an eye on these errors as more betas are released as this points to being an Xcode bug.

**Comment by salling** [comment]
I think the problem is that these import statements are missing in the AppboyKit.h umbrella header:
```
#import "ABKModalWebViewController.h"
#import "ABKNoConnectionLocalization.h"
```

**Comment by lowip** [comment]
@mbanasiewicz, @vrutings, @salling, @xav42mac

We have released [3.33.1](https://github.com/Appboy/appboy-ios-sdk/releases/tag/3.33.1) which fixes this issue. 
Thanks @salling for the help!

Best,
